.travis.yml

```diff
@@ -1,30 +1,45 @@
 language: php
 
-php:
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
-  - hhvm
+sudo: false
 
-env:
-  - SYMFONY_VERSION=2.1.*
-  - SYMFONY_VERSION=2.2.*
-  - SYMFONY_VERSION=2.3.*
-  - SYMFONY_VERSION=dev-master
+branches:
+  only:
+    - master
+
+matrix:
+  include:
+    - php: 5.3
+      env:
+        - deps=low
+        - SYMFONY_VERSION=2.7.*
```

Comment:
Indeed sorry, i'm used to libraries loading the entire framework...
That's fine for me then :+1: